packages/docusaurus-theme-classic/src/options.ts

```diff
@@ -78,6 +78,9 @@ export const DEFAULT_CONFIG: ThemeConfig = {
     minHeadingLevel: 2,
     maxHeadingLevel: 3,
   },
+  codeBlock: {
+    useLanguageAsTitle: false,
+  },
```

Comment:
CI fails because defining `DEFAULT_CONFIG` is not enough, you also have to declare a Joi schema and define defaults for `themeConfig.codeBlock` and `themeConfig.codeBlock.useLanguageAsTitle` otherwise users providing `themeConfig: {}` will not have these fields by default.